src/cogs/jaegeraccounts.py

```diff
@@ -156,30 +234,34 @@ async def book(self, ctx):
         async with dbPool.acquire() as conn:
             url = await conn.fetchval("SELECT url FROM sheet_urls WHERE fk = (SELECT id FROM guilds WHERE guild_id = $1);", ctx.guild.id)
         if url is None:
-            raise NoSheetsUrlException("There is no google sheets url associated with this guild.")
+            raise NoSheetsUrlException(
+                "There is no google sheets url associated with this guild.")
 
         sheet_data = await SheetData.from_url(self.bot, ctx, url)
 
-        # Necessary because author.nick is None if the user has not changed his name on the server      
+        # Necessary because author.nick is None if the user has not changed his name on the server
         name = ctx.author.name
         if ctx.author.nick is not None:
             name = ctx.author.nick
 
         # Try to assign accounts to the person that last had it as often as possible.
         for account in sheet_data.accounts:
             if account.last_user == name and account.is_booked:
-                await ctx.reply(f"You have already been assigned: `{account.name}`.\n"
+                if account.is_booked:
+                    await ctx.reply(f"You have already been assigned: `{account.name}`.\n"
                                     "Please check your PMs for the login details.")
-                return
-
+                    return
+                else:
+                    await ctx.author.send(embed=account.embed)
+                    return
```

Comment:
The embed already gets sent later on in the function, and account.is_booked is already checked in line 249